src/graph/visitor/PrunePropertiesVisitor.cpp

```diff
@@ -57,7 +57,7 @@ void PrunePropertiesVisitor::visitCurrent(Project *node) {
     rootNode_ = false;
     return;
   }
-  for (auto i = 0u; i < columns.size(); ++i) {
+  for (auto i = 0u; i < columns.size() && i < colNames.size(); ++i) {
```

Comment:
AFAIK, the columns size should always be same as the colNames, maybe we have to check what causes the different result here.

Reply:
I wasn't sure if these two shall always have equal sizes. If so, let me check it then.